packages/web3-utils/src/types.ts

```diff
@@ -3,3 +3,12 @@ export type ValueTypes = 'address' | 'bool' | 'string' | 'int256' | 'uint256' |
 export type Address = string;
 export type Bytes = Buffer | Uint8Array | ArrayBuffer | HexString;
 export type Numbers = number | bigint | string | HexString;
+export type EncodingTypes = Numbers | boolean | Numbers[] | boolean[];
+export type typedObject = {
```

Comment:
Always use title case. 

```suggestion
export type TypedObject = {
```